Enemies/Dodongo.cs

```diff
@@ -7,7 +7,6 @@ namespace LegendOfZelda
     public class Dodongo : IEnemy
     {
         private readonly List<AnimatedSprite> Sprites;
-        private readonly List<AnimatedSprite> HurtSprites;
```

Comment:
Taking these out seems to help overall, I think that its a good change just to get this enemy presentable.